Common/DtaStructures.h

```diff
@@ -636,7 +636,20 @@ class CScsiCmdInquiry_StandardData
     uint8_t         m_T10VendorId[8];                       //  8
     uint8_t         m_ProductId[16];                        // 16
     uint8_t         m_ProductRevisionLevel[4];              // 32
-};                                  // 36
+    uint8_t         m_DriverSerialNumber[8];                // 36
```

Comment:
This seems unused now.  And it is not a serial number.

Reply:
Removed.